cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -270,6 +271,14 @@ def _delete_entrance_exam(request, course_key):
         }
         CourseMetadata.update_from_dict(metadata, course, request.user)
 
+    # Remove any/all Entrance Exam graders from the course
+    grading_model = CourseGradingModel.fetch(course_key)
```

Comment:
Seems like this same operation is repeated within the `create_xblock` helper. Might be better to make it a helper in its own right, and call it from here, and within the other helper.